modules/is/manifests/params.pp

```diff
@@ -35,6 +35,44 @@
 
   $start_script_template = 'bin/wso2server.sh'
 
+  $template_list = [
+    'repository/conf/carbon.xml',
+    'repository/conf/user-mgt.xml',
+    'repository/conf/datasources/master-datasources.xml',
```

Comment:
Can we further look into this and decide the files which can go to the common module as well. AFAIU master-datasources like files can go to the common module?

Reply:
We are in the process of creating a list of common templates and parameters that could be included in the common module. After the list is finalized, all of these can be moved.